gruntfile.js

```diff
@@ -208,7 +208,7 @@ module.exports = function (grunt) {
   require('load-grunt-tasks')(grunt);
 
   // Making grunt default to force in order not to break the project.
```

Comment:
I think letting people override from cmd is better because the cmd is already open to run grunt anyway.